analyses/cell-type-ETP-ALL-03/renv.lock

```diff
@@ -2076,8 +2372,8 @@
       "RemoteHost": "api.github.com",
       "RemoteUsername": "AlexsLemonade",
       "RemoteRepo": "rOpenScPCA",
-      "RemoteRef": "main",
-      "RemoteSha": "fc784446f8d86b072e6f7f67287adfff598f4911",
+      "RemoteRef": "f71a8191130fa543e6506c73c7b62ffa55e8ba3f",
```

Comment:
Did you make this update manually, installing the commit directly? I would have expected the `RemoteRef` to stay as `main`, which will enable future updates with just `renv::update("rOpenScPCA")`. Otherwise we have to keep track of the hashes much more manually, which is annoying.

Reply:
I did `renv::install("rOpenScPCA@......")`. Will go again with `update`

Reply:
You should be able to just change these lines manually.
```suggestion
      "RemoteRef": "main",
```